cmd/query/app/querysvc/query_service.go

```diff
@@ -63,13 +63,13 @@ func NewQueryService(spanReader spanstore.Reader, dependencyReader dependencysto
 }
 
 // GetTrace is the queryService implementation of spanstore.Reader.GetTrace
-func (qs QueryService) GetTrace(ctx context.Context, traceID model.TraceID) (*model.Trace, error) {
-	trace, err := qs.spanReader.GetTrace(ctx, traceID)
+func (qs QueryService) GetTrace(ctx context.Context, query *spanstore.TraceIDQueryParameters) (*model.Trace, error) {
```

Comment:
This is a breaking change, we can't accept it. Need a different solution.

Reply:
Change must be backwards compatible with existing API.

Reply:
I understand. Can we simply ignore the start and end time parameters at present? otherwise it would touch lots of files.

Reply:
Sorry, I may have misread the change, it might be still OK. I don't have a problem with changing the internal APIs, as long as external change is backwards compatible, which looks like it may be.

The problem is that you're regenerating proto types, but I am not seeing a corresponding `.proto` change. We need to start there (it's in jaeger-idl repo).

Reply:
had sent the PR https://github.com/jaegertracing/jaeger-idl/pull/95. please check. @yurishkuro

Reply:
@yurishkuro please help. Thx

Reply:
sorry didn't see it - I left a comment on the IDL PR

Reply:
had updated the IDL PR. please check